### PR TITLE
Update desktop-changelog.rst

### PR DESCRIPTION
**Pull request by amyblais**
Adding notes about upcoming v4.4.1 release so users are aware.

### CODE REVIEW

File: source/help/apps/desktop-changelog.rst [hunk 1]
```diff
@@ -6,7 +6,16 @@ Release v4.4.0
...
+ - Fixing an issue where Google oAuth with Gmail addresses no longer works on the Desktop app. `MM-23057 <https://mattermost.atlassian.net/browse/MM-23057>`_
+ - Fixing an issue where the app sometimes doesn't restore to the right position but "jumps" to a different place in the taskbar when minimizing the app and then maximizing it. `MM-23195 <https://mattermost.atlassian.net/browse/MM-23195>`_
+ - Fixing an issue where users are not able to paste text into the login screen. `MM-23784 <https://mattermost.atlassian.net/browse/MM-23784>`_
+ - Fixing an issue where back/forward navigation on the OAuth window causes the app to crash. `MM-23153 <https://mattermost.atlassian.net/browse/MM-23153>`_
+ - Fixing an issue where a Desktop notification is not shown when session expires on Windows 10. `MM-23806 <https://mattermost.atlassian.net/browse/MM-23806>`_
+ - Fixing Segmentation Fault crashes that might be be experienced in some linux desktop clients. This is an upstream bug in the ``libnotifyapp`` library and a recommended workaround is to disable the system tray icon in the Desktop settings.
```
Comment (esethna) [inline source/help/apps/desktop-changelog.rst:16]: Not sure what this means to a user? "segmentation fault crashes"

File: source/help/apps/desktop-changelog.rst [hunk 2]
```diff
@@ -6,7 +6,16 @@ Release v4.4.0
 
 **Download Binaries:** `Mattermost Desktop on GitHub <https://github.com/mattermost/desktop/releases/tag/v4.4.0>`_
 
+- **v4.4.1, release day TBD**
+ - Fixing an issue where the Desktop client opens to a blank white Window when using GPO-set teams. `MM-23082 <https://mattermost.atlassian.net/browse/MM-23082>`_
+ - Fixing an issue where Google oAuth with Gmail addresses no longer works on the Desktop app. `MM-23057 <https://mattermost.atlassian.net/browse/MM-23057>`_
```
Comment (Willyfrog) [inline source/help/apps/desktop-changelog.rst:11]: this is only for plugins

File: source/help/apps/desktop-changelog.rst [hunk 3]
```diff
@@ -6,7 +6,16 @@ Release v4.4.0
...
+ - Fixing an issue where the app sometimes doesn't restore to the right position but "jumps" to a different place in the taskbar when minimizing the app and then maximizing it. `MM-23195 <https://mattermost.atlassian.net/browse/MM-23195>`_
+ - Fixing an issue where users are not able to paste text into the login screen. `MM-23784 <https://mattermost.atlassian.net/browse/MM-23784>`_
+ - Fixing an issue where back/forward navigation on the OAuth window causes the app to crash. `MM-23153 <https://mattermost.atlassian.net/browse/MM-23153>`_
+ - Fixing an issue where a Desktop notification is not shown when session expires on Windows 10. `MM-23806 <https://mattermost.atlassian.net/browse/MM-23806>`_
```
Comment (Willyfrog) [inline source/help/apps/desktop-changelog.rst:15]: I haven't yet been able to repro this, but by preventing the os from displaying notifications (and then I don't get any but that's the expected behaviour)

Comment (amyblais) [reply source/help/apps/desktop-changelog.rst]: I removed it for now and it can be added back later if a fix for is done

**Comment by amyblais** [comment]
@Willyfrog @deanwhillier Can you help review this so we can get this merged soon?

**Pull request merged**